cfgov/unprocessed/js/modules/util/assign.js

```diff
@@ -11,10 +11,9 @@
 
 
 /**
-* @param {object} JavaScript object.
+* @param {Object} object - JavaScript object.
```

Comment:
This might be a silly question, but why is {Object} capitalized and {boolean} not?